pytorch_lightning/profiler/profilers.py

```diff
@@ -136,9 +181,16 @@ def __init__(self, output_filename: str = None, line_count_restriction: float =
                 or a decimal fraction between 0.0 and 1.0 inclusive (to select a percentage of lines)
         """
         self.profiled_actions = {}
-        self.output_filename = output_filename
         self.line_count_restriction = line_count_restriction
 
+        self.output_fname = output_filename
+        self.output_file = open(self.output_fname, 'w') if self.output_fname else None
+
+        streaming_out = [log.info]
```

Comment:
i'm not sure we want to always log this report, as it is quite verbose.

Reply:
So add a argument to enable this logging?